packages/schemas/src/foundations/jsonb-types.ts

```diff
@@ -44,13 +52,84 @@ export const userLogPayloadGuard = z.object({
 
 export type UserLogPayload = z.infer<typeof userLogPayloadGuard>;
 
+/**
+ * Connectors
+ */
+
 // TODO: support empty shape of object
 export const connectorConfigGuard = z.object({});
 
 export type ConnectorConfig = z.infer<typeof connectorConfigGuard>;
 
+/**
+ * Settings
+ */
+
 export const adminConsoleConfigGuard = z.object({
   applicationSkipGetStarted: z.boolean(),
 });
 
 export type AdminConsoleConfig = z.infer<typeof adminConsoleConfigGuard>;
+
+/**
+ * SignIn Experience
+ */
+
+export const companyInfoGuard = z.object({
+  name: z.string(),
+  logo: z.string(),
+});
+
+export type CompanyInfo = z.infer<typeof companyInfoGuard>;
+
+export enum BrandingStyle {
+  companylogo_companyname_appname = 'cl-cn-an',
+  companylogo_applogo_campanyname_appname = 'cl-al-cn-an',
+  applogo_campanyname_appname = 'al-cn-an',
```

Comment:
good one. shall we use something more specific, e.g.: `CompanyLogo_CompanyName_AppName` as other enums are written in CamelCase (in `packages/schemas/src/db-entries/custom-types.ts`)?

Reply:
updated